docs/core/howto/ssl.rst

```diff
@@ -204,10 +199,40 @@ For example,
     options = CertificateOptions(..., dhParameters=dhParams)
 
 Another part of the TLS protocol which ``CertificateOptions`` can control is the version of the TLS or SSL protocol used.
-This is often called the context's "method".
+There are two ways to configure it, either by configuring a list of protocols, or by the legacy way of setting the internal context's "method".
+By default, Twisted will configure it to use TLSv1.0 or later and disable the insecure SSLv3 protocol, but this manual control over protocols can be helpful if you need to support legacy SSLv3 systems, or you wish to restrict it down to just the strongest of the TLS versions.
+
+You can give ``CertificateOptions`` a range of TLS versions you wish it to negotiate with the ``tlsProtocols`` argument in the initializer:
+
+.. code-block:: python
+
+    from twisted.internet.ssl import (CertificateOptions, TLSVersion,
+                                      TLSVersionRange)
+    options = CertificateOptions(
+        ...,
+        tlsProtocols=TLSVersionRange(TLSVersion.TLSv1_0,
+                                     TLSVersion.TLSv1_2))
+
```

Comment:
> On top of that, it means that if you have an application that you wrote in 2016 that is still being used in 2024 when a hypothetical TLSv1.4 is used, the set of "included only" TLS versions would exclude the use of TLSv1.4. 

Yes.  This is what the `GOOD_TLS_VERSIONS` constant I invented for Twisted to expose is for.  It will include TLSv1.4 in 2024 and applications that use it for `tlsProtocols` will inherit that change without any code changes.